Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
```

Comment:
@dorothyas Please untrack or delete this file

Reply:
I added this because the heroku build kept failing without this file

Reply:
okay

Reply:
I have deleted it.